tracer_options.go

```diff
@@ -144,3 +145,15 @@ func (tracerOptions) BaggageRestrictionManager(mgr baggage.RestrictionManager) T
 		tracer.baggageRestrictionManager = mgr
 	}
 }
+
+func (tracerOptions) DebugThrottler(throttler throttler.Throttler) TracerOption {
+	return func(tracer *Tracer) {
+		tracer.debugThrottler = throttler
+	}
+}
+
+func (tracerOptions) Throttler(throttler throttler.Throttler) TracerOption {
+	return func(tracer *Tracer) {
+		tracer.throttler = throttler
+	}
+}
```

Comment:
Why are we doing two throttlers upfront? I thought we would use one throttler and just allow the users to define the type of throttling to do.

Reply:
I thought we haven't even agreed that throttling regular sampling is not going to mess up xys.

Reply:
tis a proof of concept, I wanted to make sure we could throttle the sampler without being too invasive. I'll be removing this code.

Reply:
Cool. I was thinking more like the option based approach if we need it.